extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -328,7 +336,16 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   getAllMedia_() {
```

Comment:
I think this method should always return an array, and not sometimes an array, sometimes a NodeList

Reply:
Should we update ``getAllVideos_`` too? It's used to know if there's a video loading, to maybe display the loading spinner indicator.

Reply:
Changed to always return same type. Updated `getAllVideos_` with similar logic.